plugins/obs-outputs/librtmp/rtmp.c

```diff
@@ -484,7 +484,8 @@ RTMP_Reset(RTMP *r)
     r->m_fVideoCodecs = 252.0;
     r->Link.curStreamIdx = 0;
     r->Link.nStreams = 0;
-    r->Link.timeout = 30;
+    r->Link.receiveTimeout = 30;
+    r->Link.sendTimeout = 4;
```

Comment:
How was this value determined? The PR says 8 seconds but the code uses 4. I'm worried 4 seconds might be a little short to allow for proper TCP retransmission / recovery on higher latency connections such as mobile data.

Reply:
On my system (Ubuntu) the actual timeout is double whatever the configured timeout is. It seems to be a common problem which is why I set the number. Just increased it to 6 seconds since Windows probably behaves normally.